src/Util/PhpstormHelper.php

```diff
@@ -48,8 +48,11 @@ public static function handleArgvFromPhpstorm(array &$argv, string $paratestBina
     /** @param  array<int, string> $argv */
     private static function getArgvKeyFor(array $argv, string $searchFor): int
     {
+        $searchForForwardSlash = str_replace('\\', '/', $searchFor);
+        $searchForBackwardSlash = str_replace('/', '\\', $searchFor);
```

Comment:
```suggestion
        $searchFor = str_replace('/', DIRECTORY_SEPARATOR, $searchFor);
```
Instead of a double search, what about a single search with the os-aware `DIRECTORY_SEPARATOR` constant?